gfx/MShockXotto+/pngs_tiles_32x32/item/Melee/polearms.json

```
@@ -0,0 +1,11 @@
+{"id": "pike", "fg": ["pike"], "rotates": false, "bg": []},
+{"id": "pike_copper", "fg": ["pike"], "rotates": false, "bg": []},
+{"id": "pike_wood", "fg": ["pike"], "rotates": false , "bg": []},
+{"id": "halberd", "fg": ["halberd"], "rotates": false, "bg": []},
+{"id": "poleaxe", "fg": ["poleaxe"], "rotates": false, "bg": []},
+{"id": "naginata", "fg": ["naginata"], "rotates": false, "bg": []},
+{"id": "glaive", "fg": ["glaive"], "rotates": false, "bg": []},
+{"id": "brush_axe", "fg": ["brush_axe"], "rotates": false, "bg": []},
+{"id": "brush_axe_bronze", "fg": ["brush_axe_bronze"], "rotates": , "bg": []}
+{"id": "staff_pipe", "fg": ["staff_pipe"], "rotates": , "bg": []},
+{"id": "shaft_spear", "fg": ["shaft_spear"], "rotates": , "bg": []}
```

```suggestion
{
  "id": [
    "pike",
    "pike_copper",
    "pike_wood"
  ],
  "fg": "pike"
}
```
Don't need the json if `fg` and `id` are the same, unless you're also using the sprite for different id in a different file

---

Actually you have a `pike_wood` and `pike_copper` sprites so this whole file is useless, and kinda wrong since it uses pike for pike_wood and pike_copper